.upstream-tests/test/host_only/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+function(libcudacxx_add_standalone_header_test test_file_output_var header_under_test)
```

Comment:
Is it temporary to only do this testing with host compilers? Otherwise I'd think we'd still want to do this kind of thing with nvcc too.

Reply:
We should expand this to compile other TUs. Currently it's not very 'smart' I guess, but it did find a few issues.

Reply:
Poor choice of verbage. I meant other compilers and dialects.